examples/marketing/banner

```diff
@@ -0,0 +1,44 @@
+require 'bigcommerce'
+
+Bigcommerce.configure do |config|
+  config.auth = 'legacy'
+  # You will get this url when registering for an API key
+  config.url = ENV.fetch['BC_STORE_HASH']
```

Comment:
These should probably be `()`, as `fetch` is a method call.

So you can do it either as
`ENV.fetch 'BC_STORE_HASH'`
or
`ENV.fetch('BC_STORE_HASH')`